Objects/rangeobject.c

```diff
@@ -78,8 +78,12 @@ range_new(PyTypeObject *type, PyObject *args, PyObject *kw)
         return NULL;
 
     if (PyTuple_Size(args) <= 1) {
-        if (!PyArg_UnpackTuple(args, "range", 1, 1, &stop))
+        if (!PyArg_UnpackTuple(args, "range", 1, 1, &stop)) {
```

Comment:
I think it is better to get rid of `PyArg_UnpackTuple()`. Its only benefit is checking the size of the tuple and formatting an error message which is wrong. Note also that the second `PyArg_UnpackTuple()` also can produce incorrect error message.
```c
switch (PyTuple_GET_SIZE(args)) {
case 3:
    step = PyTuple_GET_ITEM(args, 2);
    /* fall through */
case 2:
    start = PyTuple_GET_ITEM(args, 0);
    stop = PyTuple_GET_ITEM(args, 1);
    ...
    break;
case 1:
    stop = PyTuple_GET_ITEM(args, 0);
    ...
    break;
case 0:
    // error
    return NULL;
default:
    // error
    return NULL;
}
```

Reply:
Thanks for the suggestion, @serhiy-storchaka!